prclz/reblock/reblock.py

```diff
@@ -0,0 +1,205 @@
+import argparse
+import os
+import time
+from pathlib import Path
+from typing import List, Tuple, Callable
+import geopandas as gpd
+import igraph
+import numpy as np
+import pandas as pd
+import tqdm
+from shapely.ops import polygonize
+from shapely.wkt import dumps
+from shapely.geometry import (LinearRing, LineString, MultiLineString,
+                              MultiPoint, MultiPolygon, Point, Polygon)
+from logging import basicConfig, debug, info, warning, error
+
+from .planar_graph import PlanarGraph
+
+def block_to_gadm(block: str) -> str:
+    """Just grabs the GADM from a block"""
+    block_rev = block[::-1]
+    idx = block_rev.index("_") + 1
+    return block[0:-idx]
+
+def add_buildings(graph: PlanarGraph, 
```

Comment:
make this a method of the `PlanarGraph` class, doesn't have much to do with reblocking, per se

Reply:
done